api/tests/integration/infrastructure/repositories/user-repository_test.js

```diff
@@ -117,6 +117,19 @@ describe('Integration | Infrastructure | Repository | UserRepository', () => {
         // then
         expect(user.email).to.equal(userInDb.email);
       });
+
+      it('should return a domain user when email case sensitive found', async () => {
+        // given
+        const email = 'USER@example.net';
+        databaseBuilder.factory.buildUser({ email });
+        await databaseBuilder.commit();
+
+        // when
+        const foundUser = await userRepository.getByEmail(email.toLowerCase());
+
+        // then
+        expect(foundUser).to.exist;
+      });
```

Comment:
**suggestion**
Je comprends ton intention: faire un test KO puis implémenter.
Mais du coup le nom du test est contradictoire: la recherche est case-insensitive
Je te propose de modifier la fixture, le libellé et l'assertion du cas de test existant et de supprimer celui que tu as ajouté
`should return a domain user when email case insensitive found`

Ca donne
```
const mixCaseEmail = 'USER@example.net';
userInDb = databaseBuilder.factory.buildUser(userToInsert);
it('should return a domain user when email match (case insensitive)', ()=>{
        // given
        const searchEmail = userInDb.email.toUpperCase();

        // when
        const user = await userRepository.getByEmail(searchEmail);

        // then
        expect(user.id).to.equal(userInDb.id);
});
```

Reply:
Done